internal/handlers/grpcsrv.go

```diff
@@ -0,0 +1,71 @@
+package handlers
```

Comment:
why is this package named handlers? This should be package `internal/grpc` to allow a generic design. Note that gRPC handlers are typically referred to as resolvers not handlers.